package.json

```diff
@@ -24,6 +24,9 @@
   },
   "homepage": "https://github.com/pelias/query#readme",
   "dependencies": {
+    "@turf/transform-scale": "^5.1.5",
+    "@turf/turf": "^5.1.6",
+    "jsondiffpatch": "^0.4.1",
```

Comment:
Is this being used, I couldn't find a corresponding `require()`?

Reply:
you're right, deleted.